src/scripts/site.js

```diff
@@ -420,586 +644,747 @@ var Dashboard = {
+            anonymous: true,
             controller: "dashboard/wizardstart"
-        }), defineRoute({
+        });
+        defineRoute({
             path: "/wizarduser.html",
             dependencies: ["dashboardcss", "emby-input"],
             controller: "scripts/wizarduserpage",
-            autoFocus: !1,
-            anonymous: !0
-        }), defineRoute({
+            autoFocus: false,
+            anonymous: true
+        });
+        defineRoute({
             path: "/videoosd.html",
             dependencies: [],
             transition: "fade",
             controller: "scripts/videoosd",
-            autoFocus: !1,
+            autoFocus: false,
             type: "video-osd",
-            supportsThemeMedia: !0,
-            fullscreen: !0,
-            enableMediaControl: !1
-        }), defineRoute({
+            supportsThemeMedia: true,
+            fullscreen: true,
+            enableMediaControl: false
+        });
+        defineRoute({
             path: "/configurationpage",
             dependencies: [],
-            autoFocus: !1,
-            enableCache: !1,
-            enableContentQueryString: !0,
+            autoFocus: false,
+            enableCache: false,
+            enableContentQueryString: true,
             roles: "admin"
-        }), defineRoute({
+        });
+        defineRoute({
             path: "/",
-            isDefaultRoute: !0,
-            autoFocus: !1,
+            isDefaultRoute: true,
+            autoFocus: false,
             dependencies: []
-        })
+        });
     }
 
     function getPluginPageContentPath() {
-        return window.ApiClient ? ApiClient.getUrl("web/ConfigurationPage") : null
+        if (window.ApiClient) {
+            return ApiClient.getUrl("web/ConfigurationPage");
+        }
+
+        return null;
     }
 
     function loadPlugins(externalPlugins, appHost, browser, shell) {
         console.log("Loading installed plugins");
-        var list = [
-            "bower_components/emby-webcomponents/playback/playbackvalidation",
-            "bower_components/emby-webcomponents/playback/playaccessvalidation",
-            "bower_components/emby-webcomponents/playback/experimentalwarnings",
-            "bower_components/emby-webcomponents/htmlaudioplayer/plugin",
-            "bower_components/emby-webcomponents/htmlvideoplayer/plugin",
-            "bower_components/emby-webcomponents/photoplayer/plugin",
-            "bower_components/emby-webcomponents/youtubeplayer/plugin"
-        ];
+        var list = ["bower_components/emby-webcomponents/playback/playbackvalidation", "bower_components/emby-webcomponents/playback/playaccessvalidation", "bower_components/emby-webcomponents/playback/experimentalwarnings", "bower_components/emby-webcomponents/htmlaudioplayer/plugin", "bower_components/emby-webcomponents/htmlvideoplayer/plugin", "bower_components/emby-webcomponents/photoplayer/plugin", "bower_components/emby-webcomponents/youtubeplayer/plugin"];
```

Comment:
It was better before
```suggestion
        var list = [
            "bower_components/emby-webcomponents/playback/playbackvalidation",
            "bower_components/emby-webcomponents/playback/playaccessvalidation",
            "bower_components/emby-webcomponents/playback/experimentalwarnings",
            "bower_components/emby-webcomponents/htmlaudioplayer/plugin",
            "bower_components/emby-webcomponents/htmlvideoplayer/plugin",
            "bower_components/emby-webcomponents/photoplayer/plugin",
            "bower_components/emby-webcomponents/youtubeplayer/plugin"
        ];
```